server/container/mailbox-adapter/src/main/java/org/apache/james/adapter/mailbox/MailboxManagerManagement.java

```diff
@@ -142,6 +148,25 @@ public void deleteMailbox(String namespace, String user, String name) {
         }
     }
 
+    @Override
+    public void importEmlFileToMailBox(String namespace, String user, String name, String emlpath) {
+        checkMailboxArguments(namespace, user, name, emlpath);
+        MailboxSession session = null;
+        try {
+            session = mailboxManager.createSystemSession(user, log);
+            mailboxManager.startProcessingRequest(session);
+            MessageManager messageManager = mailboxManager.getMailbox(new MailboxPath(namespace, user, name), session);
+            InputStream emlFileResourceAsStream = new FileInputStream(emlpath);
+            ComposedMessageId composedMessageId = messageManager.appendMessage(emlFileResourceAsStream, new Date(),
+                    session, false, new Flags());
```

Comment:
What does false means? We tend to extract values to constant to give them a name.

I believe, if it is isRecent then it should be true.